composer.json

```diff
@@ -67,7 +71,8 @@
 
 		"symfony/browser-kit": "~2.7",
 		"silex/web-profiler": "~2.0",
-		"sorien/silex-dbal-profiler": "~2.0"
+		"sorien/silex-dbal-profiler": "~2.0",
+		"wmde/fundraising-frontend-content": "dev-master as 1.0.x-dev"
```

Comment:
You are still stacking the master branch, even if you have the alias. I can see how an alias would make a difference if it is defined in the repository, but not what the inline thing here solves.